### ISSUE TITLE
Advantage keyboard navigation issues

### ISSUE THREAD

**Issue by steverydz** [canonical-web-and-design/ubuntu.com#9081]
Raised by @elioqoshi here: https://github.com/canonical-web-and-design/ubuntu.com/pull/9055#issuecomment-763729592

**Comment by SirSamTumless** [comment]
> I just noticed another issue (in 2 places) in terms of keyboard accessibility in navigation.
> 
> When selecting Servers, Other VMs or Desktops with the arrow keys, focus goes automatically to the "How many servers" section. Expected behaviour should be that the user needs to confirm with the Tab key to confirm and continue to the next section. I am unsure if this is intended, but this behavior would be worse accessibility-wise than what is already on the Ubuntu.com production site currently (you select the kind of machine you want, press Tab and the "How many servers" section is focused.
> 
> This also seems to be a bug in the current production website: After adding a subscription to your cart and selecting "Buy now", a modal with the user's payment information shows up. When trying to go back with the keyboard here (Shift+Tab) the selected focus is not the modal, but the background page elements. The background page should not be accessible without closing the modal.
> 
> Let me know if these make sense and if you need further context, I'd be happy to further illustrate these interactions.

**Comment by elioqoshi** [comment]
This seems to not be an issue anymore, although pressing anywhere 'Enter' after navigating via keyboard on the page, seems to make the Other versions modal appear.

![image](https://user-images.githubusercontent.com/5436686/120225523-ae6f6500-c245-11eb-91d2-78a3c67e1b2f.png)

@ClementChaumel any ideas why?

**Comment by wgx** [comment]
This is merged: https://github.com/canonical-web-and-design/ubuntu.com/pull/9055#issuecomment-763729592  - closing
